<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="Tab_action">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Tab_translator">
       <attribute name="title">
        <string>Translator</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QWidget" name="leftPanel" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <item>
             <widget class="QLineEdit" name="txtsearch">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tblList">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="centerPanel" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="txtOriginal"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="txtTranslation"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="rightPanel" native="true">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="BfnPreviewWidget" name="widget" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Tab_font">
       <attribute name="title">
        <string>Font</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>862</width>
                 <height>608</height>
                </rect>
               </property>
               <widget class="QLabel" name="lblFontTexture">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>63</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>thông số</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLineEdit" name="txtHexCode"/>
               </item>
               <item>
                <widget class="QSpinBox" name="spinWidth"/>
               </item>
               <item>
                <widget class="QSpinBox" name="spinAdvance"/>
               </item>
               <item>
                <widget class="QPushButton" name="btnUpdateFont">
                 <property name="text">
                  <string>Apply</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="actionExit"/>
    <addaction name="actionImport_Folder"/>
    <addaction name="actionImport_Font"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionExport_to_JSON"/>
    <addaction name="actionImport_to_JSON"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/new.png</normaloff>:/image/res/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project       ctrl+n</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/open.png</normaloff>:/image/res/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project         ctrl+o</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/save.png</normaloff>:/image/res/save.png</iconset>
   </property>
   <property name="text">
    <string>Save                    ctrl+s</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/close.png</normaloff>:/image/res/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/exit.png</normaloff>:/image/res/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/undo.png</normaloff>:/image/res/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo         ctrl+z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/cut.png</normaloff>:/image/res/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut             ctrl+x</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/copy.png</normaloff>:/image/res/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy          ctrl+c</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/paste.png</normaloff>:/image/res/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste          ctrl+v</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/redo.png</normaloff>:/image/res/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo         ctrl+y</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/res/save as.png</normaloff>:/image/res/save as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExport_to_JSON">
   <property name="text">
    <string>Export to JSON</string>
   </property>
  </action>
  <action name="actionImport_to_JSON">
   <property name="text">
    <string>Import to JSON</string>
   </property>
  </action>
  <action name="actionImport_Folder">
   <property name="text">
    <string>Import Folder</string>
   </property>
  </action>
  <action name="actionImport_Font">
   <property name="text">
    <string>Import Font</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BfnPreviewWidget</class>
   <extends>QWidget</extends>
   <header>bfnpreviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
